locations/spiders/dor_alon_il.py

```
@@ -0,0 +1,66 @@
+            for field, days in self.HOURS.items():
+                value = station.get(field)
+                if not value or value == "None":
+                    continue
+                for day in days:
+                    if value == "24":
+                        oh.add_range(day, "00:00", "24:00")
+                    elif value == "סגור":  # closed
+                        oh.set_closed(day)
+                    elif hours := re.match(r"^(\d{1,2}:\d{2})-(\d{1,2}:\d{2})$", value):
+                        oh.add_range(day, hours.group(1), hours.group(2))
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)